packages/web/server/AssetBase.ts

```diff
@@ -1,5 +1,6 @@
 import { Attachment, FieldSet, Table } from 'airtable'
 import getConfig from 'next/config'
+import AssetProps from 'src/../fullstack/AssetProps'
```

Comment:
Why this absolute and relative import?

Reply:
I should really move fullstack folder inside of the src folder.

Reply:
ill create a PR that does this